NOTES-WINDOWS.md

```diff
@@ -195,7 +195,7 @@ MinGW offers an alternative way to build native OpenSSL, by cross compilation.
 
  * It is also possible to build mingw[64] on Linux or Cygwin.
 
-   In this case configure with the corresponding `--cross-compile-prefix=`
+   In this case, configure with the corresponding `--cross-compile-prefix=`
```

Comment:
As a native English speaker, I would say that many of the `,` changes in this PR are a matter of style, rather than correctness.

Reply:
Reverted back that one